base.yaml

```diff
@@ -32,6 +50,45 @@ sensor:
     name: "Openings"
     unit_of_measurement: "openings"
     icon: mdi:open-in-app
+  - platform: ratgdo
+    id: ${id_prefix}_paired_devices_total
+    type: paired_devices_total
+    entity_category: diagnostic
+    ratgdo_id: ${id_prefix}
+    name: "Paired Devices"
+    icon: mdi:remote   
+  - platform: ratgdo
+    id: ${id_prefix}_paired_devices_remotes
+    type: paired_devices_remotes
+    entity_category: diagnostic
+    ratgdo_id: ${id_prefix}
+    name: "Paired Remotes"
+    disabled_by_default: true
+    icon: mdi:remote  
+  - platform: ratgdo
+    id: ${id_prefix}_paired_devices_keypads
+    type: paired_devices_keypads
+    entity_category: diagnostic
+    ratgdo_id: ${id_prefix}
+    name: "Paired Keypads"
+    disabled_by_default: true
+    icon: mdi:dialpad  
+  - platform: ratgdo
+    id: ${id_prefix}_paired_devices_wall_controls
+    type: paired_devices_wall_controls
+    entity_category: diagnostic
+    ratgdo_id: ${id_prefix}
+    name: "Paired Wall Controls"
+    disabled_by_default: true
+    icon: mdi:pan-vertical
+  - platform: ratgdo
+    id: ${id_prefix}_paired_devices_accessories
+    type: paired_devices_accessories
+    entity_category: diagnostic
+    ratgdo_id: ${id_prefix}
+    name: "Paired Accessories"
+    disabled_by_default: true
+    icon: mdi:cloud-cancel-outline  
```

Comment:
```suggestion
```

Looks like most of the memory is used by these, and since they are disabled by default, let's take them out. Someone who really wants them can add them in manually

Reply:
We could also have a base_esp32.yaml which includes base.yaml and added them in only if they have an esp32 in a future PR